src/main/java/pl/sda/weather/validation/LocationValidation.java

```diff
@@ -0,0 +1,26 @@
+package pl.sda.weather.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LocationValidation {
+
+    public boolean valCordinates(String name) {
```

Comment:
please rename -> valCordinates is not good method name. 
Method name should be self-descriptive